Gemfile

```diff
@@ -6,8 +6,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '2.7.7'
 
 gem 'bootsnap', '>= 1.4.4', require: false
-gem 'jbuilder', '~> 2.7'
 gem 'mysql2', '~> 0.5'
+gem 'oj'
```

Comment:
Performance optimizations for json serialization: https://github.com/ohler55/oj?tab=readme-ov-file#performance-comparisons